.platform/hooks/predeploy/03_add_keys_to_ssh_config.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# add github as host and ssh key to ssh config
+cat << EOF > /home/webapp/.ssh/config
+Host github.com
+  IdentityFile /home/webapp/.ssh/github
+  User git
+EOF
```

Comment:
nit: would it be better to change the owner of `config` file to webapp if it's created by this script? 
It's probably okay if we don't do it as the default permission should allow read by other users.

Reply:
we should change it, and as mentioned is in script 04